content/using_ec2_spot_instances_with_eks/helm_root/deploy_metric_server.md

```diff
@@ -4,15 +4,13 @@ date: 2020-03-07T08:30:11-07:00
 weight: 20
 ---
 
+
 ### Deploy the Metrics Server
 Metrics Server is a cluster-wide aggregator of resource usage data. These metrics will drive the scaling behavior of the [deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/). We will deploy the metrics server using `Helm` configured earlier in this workshop.
 
 ```
-kubectl create namespace metrics
-helm install metrics-server \
-    stable/metrics-server \
-    --version 2.10.0 \
-    --namespace metrics
+kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
```

Comment:
Jessie, I have a  few questions on this on this one: Would you mind commenting on what is driving that change ? What are the changes in the new version of metric-server ? Why not using helm ? 

Second and more importantly, this should apply as much as we can to everything we install in workshops: I don't think that we should point the workshops to "latest" (or any environment where you want deterministic behaviour !). I think this must point to a static known version of the metric-server. Otherwise we run the risk of changes in the metric-server repository that is not under our control to disrupt the workshop and things to stop working without us knowing. If this happens while you are in the middle of a session it could be quite annoying.